packages/spruce-types/src/gql/queries/UIEnhancements.ts

```diff
@@ -0,0 +1,23 @@
+import gql from 'graphql-tag'
+
+export default gql`
+	query UIEnhancements(
+		$view: String!
+		$sections: [String!]!
+		$payload: JSON
+		$organizationId: ID
+		$locationId: ID
+	) {
+		getUiEnhancements(
+			locationId: $locationId
+			organizationId: $organizationId
+			payload: $payload
+			sections: $sections
+			view: $view
+		) {
+			sections {
+				id
```

Comment:
@kengoldfarb is the plan to come back around and build out this query? Is there a way we can use fragments like george is doing on the front end so we don't get all those gql-tag "type exists" warning?

Reply:
yes